take_pic.py

```diff
@@ -0,0 +1,38 @@
+"""
+Takes pictures of the user then converts them to encodings and saves them
+
+press q to quit
+and p to take a photo then enter name
+"""
+import sys
+import pickle
+import cv2
+import face_recognition
+CAMINDEX = 0
+#get webcam
+cap = cv2.VideoCapture(CAMINDEX)
+encodings = []
```

Comment:
Since comments are best used for "why" and not "what". Perhaps you could comment here why you leave it as 2 arrays instead of using a dictionary (which was also my initial reaction)